referenceframe/worldstate_test.go

```diff
@@ -41,3 +42,34 @@ func TestWorldStateConstruction(t *testing.T) {
 	_, err = NewWorldState([]*GeometriesInFrame{NewGeometriesInFrame("", []spatialmath.Geometry{noname, unnamed})}, nil)
 	test.That(t, err, test.ShouldBeNil)
 }
+
+func TestString(t *testing.T) {
+	foo, _ := spatialmath.NewSphere(spatialmath.NewZeroPose(), 10, "foo")
+	bar, _ := spatialmath.NewSphere(spatialmath.NewZeroPose(), 5, "bar")
+	testgeo, _ := spatialmath.NewSphere(spatialmath.NewZeroPose(), 7, "testgeo")
```

Comment:
[nit] Best practice here is to assign to an error variable and call `test.That(t, err, test.ShouldBeNil)` after each function. One of the more frustrating parts of Go is how little it does natively to assist in error handling.